apps/aggiemap-angular-e2e/src/integration/mobile-layers.spec.ts

```diff
@@ -1,10 +1,372 @@
+      // close layer options
+      cy.contains('arrow_back').click()
+      // close menu
+      cy.get('tamu-gisc-backdrop').click('right', {force: true})
+      // click location of a known construction zone
+      if (size[0] === 375) {
+        cy.wait(1000)
+        cy.get('canvas').click(340, 200)
+      }
+      else if (size[0] === 768) {
+        cy.wait(1000)
+        cy.get('canvas').click(540, 300)
+      }
+      else if (size[0] === 411) {
+        cy.wait(1000)
+        cy.get('canvas').click(350, 150)
+      }
+      else if (size[0] === 360) {
+        cy.wait(1000)
+        cy.get('canvas').click(325, 115)
+      }
+      // checks if popup is visible
+      cy.wait(1000)
+      cy.get('tamu-gisc-feature-mobile-popup').should('be.visible')
+    })
+
+    it('Points of Interest', function() {
+      cy.intercept('GET', '**/MapInfo_20190529/**')
+        .as("POI")
+        cy.checkLayer('7','Points of Interest')
+        cy.checkLegend('7', 'Points of Interest')
+      cy.get('tamu-gisc-layer-list').should('be.visible')
+      // toggle POI
+      cy.contains('Points of Interest').click()
+      // close layer options
+      cy.contains('arrow_back').click()
+      // close menu
+      cy.get('tamu-gisc-backdrop').click('right', {force: true})
+      // click location of a known POI
+      if (size[0] === 375) {
+        cy.wait(2000)
+        cy.get('canvas').click(248, 215)
+      }
+      else if (size[0] === 768) {
+        cy.wait(2000)
+        cy.get('canvas').click(440, 320)
+      }
+      else if (size[0] === 411) {
+        cy.wait(2000)
+        cy.get('canvas').click(260, 175)
+      }
+      else if (size[0] === 360) {
+        cy.wait(2000)
+        cy.get('canvas').click(230, 125)
+      }
+      // checks if popup is visible
+      cy.wait(2000)
+      cy.get('tamu-gisc-feature-mobile-popup').should('be.visible')
+        .and('contain.text', 'Fish Pond')
+      // drag up on pop-up and click directions
+      cy.get('.popup').move({ x: 0, y: -415, position: 'center', force: true })
+      cy.contains('Directions To Here').should('be.visible').click()
+      cy.wait(2000)
+      // click random location to begin route
+      cy.get('canvas').trigger('mouseover').click((size[0]/2), (size[1]/2))
+      cy.get('tamu-gisc-trip-planner-directions-mobile').move({ x: 0, y: -600, position: 'center', force: true })
+      // checks if directions are displayed
+      cy.get('.directions-container').should('be.visible')
+      // check if route duration and length is displayed
+      cy.get('.directions-overview').should('be.visible')
+      // slide directions back down to access other route options
+      cy.get('tamu-gisc-trip-planner-directions-mobile').move({ x: 0, y: 600, position: 'center', force: true })
+
+      // test bike route option
+      cy.wait(2000)
+      cy.get('.travel-modes > :nth-child(2) > div')
+          .click()
+          .should('have.class', 'active')
```

Comment:
As mentioned here, routing testing is totally unnecessary here and especially for many map features